tests/src/python/test_provider_oracle.py

```diff
@@ -147,6 +147,13 @@ def uncompiledFilters(self):
         ])
         return filters
 
+    def testAddFeatureWrongGeomType(self):
+        """
+        We override this test for Oracle provider, because Oracle DBs don't care
+        about geometry type constraints
```

Comment:
Come on, I hope so! If I am paying for something you don't want it to complain but just do what I want!

Reply:
?

Reply:
I hope they disabled not null constraints too ;)

Reply:
To clarify - this test was failing, because it checks to see that providers reject features which have been added with a non-matching geometry type (the Oracle provider doesn't reject these). But since this is valid for Oracle tables I don't want to modify the provider here.

Reply:
It was just a little rant, never mind ;)

Reply:
Working with Oracle does tend to invite insanity...